sriov/tests/SR_IOV_QinQ/test_SR_IOV_QinQ.py

```diff
@@ -16,6 +23,9 @@ def test_SR_IOV_QinQ(dut, trafficgen, settings, testdata):
     inside_tag = 20
     pf = settings.config["dut"]["interface"]["pf1"]["name"]
 
+    if "xxv710" in get_nic_model(dut, pf).lower():
+        pytest.xfail("Expected failure - XXV710 NICs fail QinQ.")
```

Comment:
I wonder if we should use skip rather xfail here. It depends on Intel's plan on xxv710, if Intel plans to support QinQ for 710, then xfail; otherwise skip.

@PatrickKutch Here is what we need info from you on the QinQ support for xxv710